templates/c2s-gp-finder/0/rancher-compose.yml

```diff
@@ -11,6 +11,12 @@ catalog:
       required: true
       default: "latest"
       type: "string"
+    - variable: "gp_finder_db_docker_image_tag"
```

Comment:
@st3v3nhunt is right. If you are using the profiles db then the variable will need to be called `profiles_db_docker_image_tag` for the CI scripts to work. 

If we change to having a _gp-finder-db_ repo with the Dockerfile extending the profiles db image to add the geospatial index we will use the `gp_finder_db_docker_image_tag` form. 

See the readme in [PR4](https://github.com/nhsuk/ci-deployment/pull/4) for more details on the naming conventions we currently employ.